README.md

```diff
@@ -18,164 +16,67 @@ GEI-CLI is continuing to expand what it can support. However, it supports the fo
-### Command line
-To get started you'll need to download the official [GitHub CLI](https://cli.github.com). You can run `gh extension install github/gh-gei` to install the GEI CLI. Once installed, run `gh gei --help` to learn about the options.
-
-```
-gh-gei
-  CLI for GitHub Enterprise Importer.
-
-Usage:
-  gh-gei [options] [command]
-
-Options:
-  --version       Show version information
-  -?, -h, --help  Show help and usage information
-
-Commands:
-  generate-script       Generates a migration script. This provides you the ability to review the steps that this tool will take, and optionally modify the script if desired before running it.
-  grant-migrator-role   Allows an organization admin to grant a USER or TEAM the migrator role for a single GitHub organization. The migrator role allows the role assignee to perform migrations into the target organization.
-                        Note: Expects GH_PAT env variable to be set.
-  migrate-repo          Invokes the GitHub APIs to migrate the repo and all repo data.
-  revoke-migrator-role  Allows an organization admin to revoke the migrator role for a USER or TEAM for a single GitHub organization. This will remove their ability to run a migration into the target organization.
-                        Note: Expects GH_PAT env variable to be set.
-```
-
-To generate a migration script, you'll need to set `GH_PAT` as an environment variable for your destination and `GH_SOURCE_PAT` for your source location. 
-
-## GitHub Enterprise Server (GHES) to GitHub Migration Usage
+## Using the GEI CLI
+There are 2 separate CLI's that we ship:
+- `ado2gh` - Intended for migrations from Azure DevOps -> GitHub
+- `gh gei` - Intended for GitHub -> GitHub migrations
 
-General usage will use the `generate-script` command to create a script that can be used to migrate all repositories from a GHES organization.
+To use `ado2gh` download the latest version from the [Releases](https://github.com/github/gh-gei/releases/latest) in this repo.
 
-Due to many GHES instances containing firewall rules restricting the direct API access that GEI needs to perform a migration, the GEI CLI uploads the migration's archive data to Azure Blob Storage as a step prior to starting the migration using GEI. This allows GEI to perform a migration without directly accessing the GHES instance. To get started you'll need to setup an [Azure storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal). Then, you'll pass in a [connection string](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/api/connection-strings/storage-connection-strings) to the CLI for that Azure storage account, specified in the parameters below. GEI CLI will take care of the rest.
+To use `gh gei` first install the latest [GitHub CLI](https://github.com/cli/cli#installation), then run the command
```

Comment:
Any references to how to use the `gh gei` cli should be removed and just tell folks to read the docs. We should only list ado2gh extra commands here.

Reply:
I'd rather not delete the small sections on usage, I scaled them way back from like 10 pages to 1 page, but I think we should have some extremely minimal "here's how you use this, see the docs for more" type section in the README. I feel that's expected in a typical OSS readme.

I was just trying to basically say "install it from here, run generate-script, then run the script. See the docs for more"